UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/NMDataService.cs

```diff
@@ -188,31 +188,63 @@ public async Task<YearWeekResponseDataModel> GetAllYearWeek(string correlationId
             return new YearWeekResponseDataModel { YearWeekModel = yearWeekModelList, IsYearAndWeekAttributesCached = isCached };
         }
 
-        public async Task<List<ShowDailyFilesResponseModel>> GetDailyBatchDetailsFiles(string correlationId)
+        public async Task<ShowDailyFilesResponseListModel> GetDailyBatchDetailsFiles(string correlationId)
         {
             try
             {
-                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(_userService.IsDistributorUser, correlationId);
-
-                _logger.LogInformation(EventIds.ShowDailyFilesResponseStarted.ToEventId(), "Maritime safety information request to get daily NM files response started for daily user:{SignInName} and Identity:{userId} with _X-Correlation-ID:{correlationId}", _userService.SignInName ?? "Public", _userService.UserIdentifier, correlationId);
+                BatchSearchResponse searchResult = new();
+                bool isCached = false;
+                const string frequency = "daily";
```

Comment:
Doesn't make a huge difference but in the SRS the frequencies are capitalised:
```suggestion
                const string frequency = "Daily";
```